src/System.ComponentModel.Composition/ref/project.json

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
Actually good point, since we moved the reference assemblies now to be net46 then I believe they are not needed anymore, I'll remove them.